src/utils.py

```diff
@@ -0,0 +1,167 @@
+"""Utility file for containers worker."""
+
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import subprocess
+
+from openrelik_worker_common.file_utils import OutputFile
+
+CE_BINARY = "/opt/container-explorer/bin/ce"
+
+
+def log_entry(log_file: OutputFile, message: str) -> None:
+    """Appends logs line to a log file.
+
+    Args:
+        log_file: log file.
+        message: log message.
+    """
+    with open(log_file.path, "a", encoding="utf-8") as log_writer:
+        log_writer.write(message)
+        log_writer.write("")
+
+
+def mount_disk(image_path: str, mount_point: str) -> str:
+    """Mounts disk image and returns the mount point.
+
+    Args:
+        image_path: Disk image path.
+        mount_point: Path to mount the disk.
+
+    Returns:
+        Mount point or None
+    """
+    # Assumes the disk is ext4
```

Comment:
mount should be smart enough to detect other filesystem types and mount them.

Reply:
I will submit another PR to identify the filesystem type and use the appropriate mount command.